scripts/ipfs/nodeinfo-ipfs

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
```

Comment:
`.sh`?

Reply:
bug in run-parts that does not executable .sh files.